jmxtrans-core/src/main/java/com/googlecode/jmxtrans/model/ResultFixtures.java

```diff
@@ -55,14 +55,28 @@ public static Result booleanFalseResult() {
 	}
 
 	public static Result numericResult() {
+		return numericResult(10);
+	}
+	public static Result numericResult(Object numericValue) {
 		return new Result(
 				0,
 				"ObjectPendingFinalizationCount",
 				"sun.management.MemoryImpl",
 				"ObjectDomainName",
-				"ObjectPendingFinalizationCount",
+				"MemoryAlias",
```

Comment:
Ok, why not... There is still a failing test related to this rename in `StatsDWriterFactoryIT`.